Gopkg.lock

```diff
@@ -888,6 +904,7 @@
     "github.com/codelingo/lingo/service",
     "github.com/codelingo/lingo/service/grpc",
     "github.com/codelingo/lingo/vcs",
+    "github.com/codelingo/rectangle",
```

Comment:
Can you look into why we're importing github.com/codelingo/rectangle ? cheers

Reply:
@rstorr Yeah, it's part of Leila's typed nil tenet. It's used as an example.